### ISSUE TITLE
Incubator Deployment Instructions

### ISSUE THREAD

**Issue by chukalicious** [hackforla/incubator#11]
### Dependency
- [ ] All active projects are moved to incubator

### Overview
We need to make a document that has step by step instructions for ops team members, so that new people know what to do to deploy projects.

- [ ] Review ~~Getting Started~~ Starting your own Incubator
- [ ] Review Installation Instructions

### Resources/Instructions

- [Starting your own Incubator](https://github.com/hackforla/incubator/blob/main/documentation/Starting-Your-Own-Incubator.md)
- [Installation-Instructions](https://github.com/hackforla/incubator/blob/main/documentation/Installation-Instructions.md)

**Comment by gjagnoor** [comment]
I don't think this is done and if this issue is not currently being worked on, I would love to work on it. Step by step instructions on how to deploy a project once it's ready would be good for everyone. I can make use of the tagged resources.

**Comment by ethanstrominger** [comment]
See https://github.com/hackforla/incubator/issues/11

**Comment by ethanstrominger** [comment]
@gjagnoor Are you still interested in documenting this?  I am setting it up for web client.

**Comment by fyliu** [comment]
I'm in the process of doing setup for incubator and will update the instructions in the [incubator instructions](https://github.com/hackforla/incubator/wiki) in the wiki.

**Comment by ale210** [comment]
This will be covered by [#132](https://github.com/hackforla/ops/issues/132)
